### DISCUSSION TITLE
how to setup nvchad without starter?

### DISCUSSION THREAD

**Discussion by Sumanthbhumi** [NvChad/NvChad#3038] [General]
how do i setup nvchad as config rather than as a plugin, so i can change some of the plugin which comes with the nvchad and also want to change key binds totally, i already have my custom key-binds and don't want to change them

**Comment by siduck** [comment]
please read the plugins doc at nvchad.com, you can override plugins within starter itself. Same with mappings, just dont call nvchad ones if you dont want them, or use vim.keymap.set to override or set new?
